pkgs/top-level/daw-plugins.nix

```diff
@@ -0,0 +1,48 @@
+# DAW plugins
+# ===========
+#
+# This file contains various DAW plugins, e.g. those for LV2.
+#
+
+{ lib, pkgs }:
+
+lib.makeExtensible (self: with self; let
+
+  callPackage = pkgs.newScope self;
+
+in {
+
+  inherit callPackage;
+
+  CharacterCompressor = callPackage ../applications/audio/magnetophonDSP/CharacterCompressor { };
+  CompBus = callPackage ../applications/audio/magnetophonDSP/CompBus { };
+  ConstantDetuneChorus  = callPackage ../applications/audio/magnetophonDSP/ConstantDetuneChorus { };
+  faustCompressors =  callPackage ../applications/audio/magnetophonDSP/faustCompressors { };
+  LazyLimiter = callPackage ../applications/audio/magnetophonDSP/LazyLimiter { };
+  MBdistortion = callPackage ../applications/audio/magnetophonDSP/MBdistortion { };
+  pluginUtils = callPackage ../applications/audio/magnetophonDSP/pluginUtils  { };
+  RhythmDelay = callPackage ../applications/audio/magnetophonDSP/RhythmDelay { };
+  VoiceOfFaust = callPackage ../applications/audio/magnetophonDSP/VoiceOfFaust { };
+  shelfMultiBand = callPackage ../applications/audio/magnetophonDSP/shelfMultiBand  { };
```

Comment:
I'd keep them in a set (`magnetophonDSP` with `recurseIntoAttrs`), otherwise their names seem too generic (`pluginUtils`).

Reply:
I should have chosen a better name for that repo / those plugins.
As it stands there is no name-clash (yet) in daw-plugins so we could leave it like this.

I would suggest to put all plugins in ``applications/audio/daw-plugins/``, and get rid of the magnetophonDSP subdir and namespace.

Reply:
OK.